src/app/components/image/image.interface.ts

```diff
@@ -29,4 +29,9 @@ export interface ImageTemplates {
      * Custom template of closeicon.
      */
     closeicon(): TemplateRef<any>;
+
+    /**
+     * Custom template of imagedescription.
```

Comment:
space here? `image description`

Reply:
@SoyDiego It is a name of the template which is imagedescription (without space).
I made it the same way as comments above (closeicon, rotaterighticon etc). Do you still want me to put a space there?

Reply:
oh ok, sorry, thanks for re check!
Now we should wait if PrimeNG Team approve it!